app/stac_api/collection_views.py

```diff
@@ -1,17 +1,27 @@
 from rest_framework import generics
-
+from rest_framework.pagination import PageNumberPagination
 from django.shortcuts import get_object_or_404
-
 from stac_api.models import Collection
 from stac_api.serializers import CollectionSerializer
 
 
+# using this extra pagination class for collection list view allows for having
+# global pagination settings in `settings_prod.py` that apply generally and to
+# override those settings for specific views if needed.
+# TODO The very small value of 2 is chosen here for testing, as we only have a very #pylint: disable=fixme
+# small number of collections in the test db so far. In the end, this value has
+# to be set to something more meaningful.
+class CollectionPagination(PageNumberPagination):
+    page_size = 2
```

Comment:
I would remove this and set the pagination in the `settings_dev.py` as follow:
```python
if DEBUG:
    REST_FRAMEWORK['PAGE_SIZE'] = os.getenv(PAGE_SIZE, 2)
```
This avoid the TODO im code and being able to reduce the page size even once we are on prod can always be useful for development where we won't develop using the actual data. It also gives the possibility to change the page size by environment variable..

Reply:
:+1:

Reply:
done